diff --git a/.gitignore b/.gitignore
index 2bfe577..c5cc104 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,4 +1,2 @@
 *.o
-config.h
-patches.h
 st
diff --git a/config.mk b/config.mk
index ffcd3cb..53d2f4d 100644
--- a/config.mk
+++ b/config.mk
@@ -31,7 +31,7 @@ PKG_CONFIG = pkg-config
 #SIXEL_LIBS = `$(PKG_CONFIG) --libs imlib2`

 # Uncomment for the netwmicon patch / NETWMICON_PATCH
-#NETWMICON_LIBS = `$(PKG_CONFIG) --libs gdlib`
+NETWMICON_LIBS = `$(PKG_CONFIG) --libs gdlib`

 # includes and libs, uncomment harfbuzz for the ligatures patch
 INCS = -I$(X11INC) \
